### DISCUSSION TITLE
为什么我无法申请学生包

### DISCUSSION THREAD

**Discussion by steve12311** [community/community#147966] [GitHub Education]
### Hi everyone,

我点击申请后会跳转到组织页面，无法填写信息申请
![屏幕截图 2024-12-26 224858](https://github.com/user-attachments/assets/c4e69b97-c2bb-445a-9c36-d07ec2f90d12)
![屏幕截图 2024-12-26 224746](https://github.com/user-attachments/assets/249d941a-ebf0-4476-9bcf-0050824807c0)

**Comment by ebndev** [comment]
Hi @steve12311, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
